src/person.rb

```diff
@@ -1,4 +1,4 @@
-require 'nameable'
+require './nameable'
```

Comment:
- [ ] The correct way to invoke a file in the same directory is using `require_relative` instead of `require` it is used to load modules or libraries that are globally installed on the system or are available in Ruby's module search path. Please 🙏🏽 use `require_relative` instead of `require` to call the app class.